.github/workflows/ci.yml

```diff
@@ -97,6 +121,23 @@ jobs:
         run: |
           cd docs && make -o genclidocs localcheck
 
+      - name: Build docs
+        env:
+          CORSO_VERSION: ${{ needs.SetEnv.outputs.version }}
+        run: |
+          cd docs &&
+            npm ci &&
+            npm run build
+
+      - uses: actions/upload-artifact@master
+        # uncomment next line to disable creating docs artifacts on PR runs
+        # if: startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/main'
```

Comment:
@meain - should we do uncomment?

Reply:
I was kinda undecided on that. As it does not get pushed anywhere for PRs, I thought having this in the artifacts will let us validate the docs that was built in CI in case we wanted to.

Reply:
I see - let's keep it